src/labels/CustomLabels.labels

```diff
@@ -6065,6 +6065,30 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Failure message if an invalid base64 url was provided</shortDescription>
         <value>The base64url format is invalid. It can only contain these characters: [a-z A-Z 0-9 _ -]</value>
     </labels>
+    <labels>
+        <fullName>gePaymentProcessErrorGeneric</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>General failure message description for tokenization or purchase errors</shortDescription>
+        <value>There was an error processing the transaction. Please review the error and try again. Alternatively, select Do Not Charge Card to enter a pledged donation without processing a card payment.</value>
```

Comment:
@dfuller114 @ptewson-sfdo Adding this as a comment instead of a suggestion because it seems like it'll need a little more  finessing. After reviewing this PR with Grace, we'd like to collapse gePaymentServicesErrorResponse into this error so it reads:

There was an error processing the payment. Please review the error and try again. Alternatively, select Do Not Charge Card to enter a pledged donation without processing a card payment. {0} errors:
- {1}

{0} = commonPaymentServices
{1} = the error coming from Payment Services.

The errors coming from Payment Services should show up as a bulleted list.

Reply:
added nested label